packages/react-theme-provider/src/createDefaultTheme.ts

```diff
@@ -54,6 +54,53 @@ export const defaultTokens: Tokens = {
       contentColor: 'var(--acent-pressed-contentColor)',
     },
   },
+  ghost: {
```

Comment:
there is also a V7 -> token mapping in `getTokens.ts` which we need to update as well